angular/src/lib/tabs/tab.component.ts

```diff
@@ -4,7 +4,13 @@ import { TabsService } from './tabs.service';
 @Component({
   selector: 'cui-tab',
   template: `
-    <a href='javascript:void(0)' [attr.role]="role" (click)='whenPress.emit()' (keydown)='whenKeyDown.emit()' >
+    <a href='javascript:void(0)'
+      [attr.role]="role"
+      (click)='whenPress.emit()'
+      (keydown)='whenKeyDown.emit()'
+      [tabIndex]='ifFocus ? 0 : -1'
```

Comment:
Will [tabIndex] work in angular? Thought it would be [attr.tabindex]

Reply:
I agree with @daanishnasir